client/grpc/cmtservice/autocli.go

```diff
@@ -0,0 +1,69 @@
+package cmtservice
+
+import (
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+	cmtv1beta1 "cosmossdk.io/api/cosmos/base/tendermint/v1beta1"
+)
+
+var CometBFTAutoCLIDescriptor = &autocliv1.ServiceCommandDescriptor{
+	Service: cmtv1beta1.Service_ServiceDesc.ServiceName,
+	RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+		{
+			RpcMethod: "GetNodeInfo",
+			Use:       "node-info",
+			Short:     "Query the current node info",
+		},
+		{
+			RpcMethod: "GetSyncing",
+			Use:       "syncing",
+			Short:     "Query node syncing status",
+		},
+		{
+			RpcMethod: "GetLatestBlock",
+			Use:       "block-latest",
+			Short:     "Query for the latest committed block",
+		},
+		{
+			RpcMethod:      "GetBlockByHeight",
+			Use:            "block-by-height [height]",
+			Short:          "Query for a committed block by height",
+			Long:           "Query for a specific committed block using the CometBFT RPC `block_by_height` method",
+			PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "height"}},
+		},
+		{
+			RpcMethod: "GetLatestValidatorSet",
+			Use:       "validator-set",
+			Alias:     []string{"validator-set-latest", "comet-validator-set", "cometbft-validator-set", "tendermint-validator-set"},
+			Short:     "Query for the latest validator set",
+		},
+		{
+			RpcMethod:      "GetValidatorSetByHeight",
+			Use:            "validator-set-by-height [height]",
+			Short:          "Query for a validator set by height",
+			PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "height"}},
+		},
+		{
+			RpcMethod: "ABCIQuery",
+			Skip:      true,
+		},
+	},
+}
+
+func NewCometBFTCommands() *cometModule {
+	return &cometModule{}
+}
+
+type cometModule struct{}
+
+func (m cometModule) IsOnePerModuleType() {}
+func (m cometModule) IsAppModule()        {}
```

Comment:
this seems confusing, can you add a godoc why this interface is met here?

Reply:
Updated at line 52, what do you think?

I'm planing to add docs on that as well.